services/core-api/app/api/mines/response_models.py

```diff
@@ -241,6 +245,7 @@ def format(self, value):
         'exemption_fee_status_note': fields.String,
         'site_properties': fields.List(fields.Nested(MINE_TYPE_MODEL)),
         'permit_prefix': fields.String,
+        'preamble_text': fields.String
     })
```

Comment:
this only exists on the permit_amendment, right? not needed on the permit

Reply:
No needed, just in permit_amendment response model.